setup.py

```diff
@@ -15,7 +15,7 @@
     use_scm_version={
         "version_scheme": "guess-next-dev",
         "local_scheme": "dirty-tag",
-        "write_to": "tiledbimg/version.py",
+        "write_to": "tiledb/version.py",
```

Comment:
Is it `tiledb/version.py` or `tiledb/bioimg/version.py` mentioned in gitignore?

Reply:
Nice catch!